packages/flutter/test/material/icons_test.dart

```diff
@@ -53,4 +55,48 @@ void main() {
       TargetPlatform.linux,
     }),
   );
+
+  testWidgets('A sample of icons look as expected', (WidgetTester tester) async {
+    await _loadIconFont();
+
+    await tester.pumpWidget(MaterialApp(
+      home: IconTheme(
+        data: const IconThemeData(size: 200),
+        child: Wrap(
+          children: const <Icon>[
+            Icon(Icons.ten_k),
+            Icon(Icons.ac_unit),
+            Icon(Icons.local_taxi),
+            Icon(Icons.local_taxi_outlined),
+            Icon(Icons.local_taxi_rounded),
+            Icon(Icons.local_taxi_sharp),
+            Icon(Icons.zoom_out_sharp),
+          ],
+        ),
+      ),
+    ));
+
+    await expectLater(find.byType(Wrap), matchesGoldenFile('test.icons.sample.png'));
+  }, skip: isBrowser); // uses dart:io
+}
+
+// Loads the Material icon font. Only necessary for golden tests.
+Future<void> _loadIconFont() async {
+  final Directory flutterRoot = Directory(path.dirname(Platform.script.toFilePath())).parent.parent;
+  final File iconFont = File(path.joinAll(<String>[
```

Comment:
We may need to revert this. This assumes that the file exists. I just saw this fail locally for me on an updated branch because the file does not exist.

Reply:
I'll revert, and rework a more robust solution